# Patch written by Toon Verstraelen <Toon.Verstraelen@UGent.be>.
# This fix is included in LibXC 2.0.2
--- libxc-2.0.1/src/lda_x.c	2013-05-08 12:54:48.922318545 -0400
+++ libxc-2.0.1-orig/src/lda_x.c	2013-05-08 12:54:07.828036433 -0400
@@ -175,7 +175,7 @@
 
   if(p->cam_omega == 0.0){
     fa_u = fa_d = 1.0;
+    a_cnst = 0.0;
-
   }else{
     a_cnst = CBRT(4.0/(9.0*M_PI))*p->cam_omega/2.0;
 
